src/service.js

```diff
@@ -368,6 +404,79 @@ function getServices(holder) {
   return services;
 }
 
+
+/**
+ * Whether the specified service implements `Disposable` interface.
+ * @param {!Object} service
+ * @return {boolean}
+ */
+export function isDisposable(service) {
+  return service.dispose !== undefined;
+}
+
+
+/**
+ * Asserts that the specified service implements `Disposable` interface and
+ * typecasts the instance to `Disposable`.
+ * @param {!Object} service
+ * @return {!Disposable}
+ */
+export function assertDisposable(service) {
```

Comment:
is it possible to move this to `Log`. not sure if it makes sense for it to reside in there but im guessing we want `dev().assertDisposable` to be removable on prod binary